src/System.Memory/tests/Performance/Perf.ReadOnlySequence.Slice.cs

```diff
@@ -2,765 +2,362 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Memory.Tests;
 using System.MemoryTests;
 using Microsoft.Xunit.Performance;
 using Xunit;
 
 namespace System.Buffers.Tests
 {
-    public class Perf_ReadOnlySequence_Slice
+    public abstract class Perf_ReadOnlySequence_Slice_Byte: Perf_ReadOnlySequence_Slice<byte>
```

Comment:
nit: Please split this into a separate cs file.

Reply:
> nit: Please split this into a separate cs file.

I do not think that it will be usefully. See for example next test files:
ReadOnlySequenceFactory.byte.cs
ReadOnlySequenceFactory.char.cs
ReadOnlySequenceTests.byte.cs
ReadOnlySequenceTests.char.cs

I used the same solution.

Reply:
OK, that is fine given the file size isn't too large (it was much larger before with but your clean up fixed that).